torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -1365,6 +1365,28 @@ void initNvFuserPythonBindings(PyObject* module) {
       py::arg("dim"),
       py::arg("index"),
       py::return_value_policy::reference);
+  nvf_ops.def(
```

Comment:
Looks about right to me. But the python API is not tested. Can we also add tests for this?

Reply:
Sorry, I do not know what to test here and where is test file for `nvf_ops.def`.